source/MaterialXCore/Interface.h

```diff
@@ -147,6 +147,29 @@ class PortElement : public ValueElement
         return getAttribute(OUTPUT_ATTRIBUTE);
     }
 
+    /// @}
+    /// @name Channels
+    /// @{
+
+    /// Set the channels string of this element, defining a channel swizzle
+    /// that will be applied to this port.
```

Comment:
Maybe elaborate and say _"...that will be applied to the upstream result if this port is connected"_.